Makefile

```diff
@@ -142,6 +145,6 @@ test-e2e-1.25: container-build-single-platform install-tools
 test-e2e-1.24: container-build-single-platform install-tools
 	$(call TEST_KUBERNETES,v1.24.0,$(PREFIX),$(VERSION)-$(PLATFORM_TAG))
 
-test-e2e-all: test-e2e-1.27 test-e2e-1.26 test-e2e-1.25 test-e2e-1.24
+test-e2e-all: test-e2e-1.28 test-e2e-1.27 test-e2e-1.26 test-e2e-1.25 test-e2e-1.24
```

Comment:
We can remove test-e2e-1.24 since we only want to run n-4 versions. Also to reduce GH actions run time as these E2E tests do take some time to complete.

Reply:
That makes sense. Removed `test-e2e-1.4`.